python/dgl/graphbolt/__init__.py

```diff
@@ -9,33 +9,36 @@
 pattern resulting in better memory usage for minibatch GNN training workloads.
 See https://pytorch.org/docs/stable/notes/cuda.html#optimizing-memory-usage-with-pytorch-cuda-alloc-conf,
 and set the environment variable `PYTORCH_CUDA_ALLOC_CONF=expandable_segments:False`
-if you want to disable it.
+if you want to disable it and set it True to acknowledge and disable the warning.
 """
 cuda_allocator_env = os.getenv("PYTORCH_CUDA_ALLOC_CONF")
-if cuda_allocator_env is None:
-    os.environ["PYTORCH_CUDA_ALLOC_CONF"] = "expandable_segments:True"
-    gb_warning(CUDA_ALLOCATOR_ENV_WARNING_STR)
-else:
-    configs = {
+WARNING_STR_TO_BE_SHOWN = None
```

Comment:
```suggestion
warning_string = None
```

Reply:
Linting was not allowing me to do this change.

Reply:
Weird, the linter didn't complain about this change now. I wonder what was different before.

Reply:
```
************* Module dgl.graphbolt
python/dgl/graphbolt/__init__.py:15:0: C0103: Constant name "warning_string" doesn't conform to UPPER_CASE naming style (invalid-name)
python/dgl/graphbolt/__init__.py:26:8: C0103: Constant name "warning_string" doesn't conform to UPPER_CASE naming style (invalid-name)
python/dgl/graphbolt/__init__.py:38:4: C0103: Constant name "warning_string" doesn't conform to UPPER_CASE naming style (invalid-name)
```